Kickstarter-iOS/Features/Comments/Views/Cells/CommentCell.swift

```diff
@@ -44,6 +45,10 @@ final class CommentCell: UITableViewCell, ValueCell {
     self.bindViewModel()
 
     self.replyButton.addTarget(self, action: #selector(self.replyButtonTapped), for: .touchUpInside)
+    self.commentCellHeaderStackView.addGestureRecognizer(UITapGestureRecognizer(
```

Comment:
I wonder if a long press would be a better interaction? But that's really gotta be a question for a new designer...

Reply:
I considered this, but since users are already used to tapping a creator's icon on project pages as a regular tap I chose to go with what's familiar.